packages/store/types/@ember/runloop/index.d.ts

```diff
@@ -1,2 +1,4 @@
-
-export const run: any;
\ No newline at end of file
+// necessary because our "run" is run.backburner
+// which we use to avoid autorun triggering for Ember <= 3.4
+// we can drop this and use run directly ~11/1/2019
+export const run: any;
```

Comment:
I *think* we *should* be able to put these in a types-only `types` package in the `packages` dir instead of repeating them in each package? But cc. @mike-north who has worked with the monorepo structure more than I have.

Reply:
I've been contemplating moving many of our private interfaces and types into a private package. Needing to know whether this would break compile is what has held me back. Publishing a package, even if technically "private" is a non-starter.

Reply:
(fwiw I've figured out that it seems its ok, so this is an option to explore, but a single root tsconfig lets us put these in one place as well so there's more than one path forward)

Reply:
to be clear, I think @chriskrycho is proposing a user-defined type root (`paths: { "*": ["foo"] }`) and not a published package. Sounds like a good suggestion to me, particularly in that there are quite a few private types floating around, and a centrally-defined single place for "non-exported things necessary for ember-data to internally compile" could catch some inconsistencies between package boundaries.

Reply:
Yep, exactly that! Safely lets us define type-only stuff in one place with zero worry we’ll accidentally publish it as a package.